config/initializers/catalog_inventory_listener.rb

```diff
@@ -1,9 +1,6 @@
 # Be sure to restart your server when you modify this file.
 
 unless defined?(::Rails::Console)
-  queue_host = ENV["QUEUE_HOST"] || "localhost"
-  queue_port = ENV["QUEUE_PORT"] || 9092
-
-  catalog_inventory_listener = CatalogInventory::EventListener.new(:host => queue_host, :port => queue_port)
+  catalog_inventory_listener = CatalogInventory::EventListener.new(:host => ClowderConfig.queue_host, :port => ClowderConfig.queue_port)
```

Comment:
@hsong-rh This file could load before the clowder_config.rb is called.

Reply:
Added `require_relative` to initialize `ClowderConfig`, if it's called first.